src/databases/Uintah/avtUintahFileFormat.C

```diff
@@ -299,7 +295,7 @@ avtUintahFileFormat::avtUintahFileFormat(const char *filename,
   if (!libHandle)
   {
 #ifdef UINTAH_LIBRARY_DIR
-  const char * lib_path = STR( UINTAH_LIBRARY_DIR );
+    const char * lib_path = UINTAH_LIBRARY_DIR;
```

Comment:
It looks like you added a space to this line.

Reply:
That one is okay